Source/Blazorise/Components/Button/Button.cs

```diff
@@ -275,7 +274,7 @@ private void OnThemeChanged( object sender, EventArgs eventArgs )
     /// <summary>
     /// Gets the size based on the theme settings.
     /// </summary>
-    protected Size ThemeSize => Size ?? Theme?.ButtonOptions?.Size ?? Blazorise.Size.Default;
+    protected Size ThemeSize => ( Size ?? ParentAddons?.Size ).GetValueOrDefault( Theme?.InputOptions?.Size ?? Blazorise.Size.Default );
```

Comment:
Could this be `.GetValueOrDefault( ParentAddons?.Size ?? Theme?.InputOptions?.Size ?? Blazorise.Size.Default )`